blog-service/2023-05-24-apps_rapid7.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Rapid7 (Apps)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - apps
+  - app catalog
+  - saas cloud
+  - symantec
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're excited to announce the release of the new Rapid7 app for Sumo Logic. This app leverages Sumo Logic [Cloud-to-Cloud Rapid7 source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/rapid7-source/) to gain deeper insights into asset and vulnerability management activities and prioritize remediation efforts, to minimize attack surfaces, and make well-informed decisions to enhance your overall security posture.
```

Comment:
```suggestion
We're releasing a new app called Rapid7 app that leverages Sumo Logic [Cloud-to-Cloud Rapid7 source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/rapid7-source/). It enables you to obtain a deeper understanding of asset and vulnerability management activities, prioritize remediation efforts, reduce attack surfaces, and make informed decisions to improve your overall security posture.
```

Reply:
I've rephrased the para to match the user-friendly tone.

Reply:
Thanks @JV0812 . Updated accordingly.